worlds/stardew_valley/test/TestMods.py

```diff
@@ -0,0 +1,66 @@
+from . import setup_solo_multiworld
```

Comment:
For some reason, your tests do not register themselves in the same way ads the existing ones.
![image](https://github.com/agilbert1412/Archipelago/assets/16685738/ec47b4c3-9ec0-4325-8c9b-3e592596962c)